<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-25T16:04:26Z" name="Suite" finished-at="2024-03-25T16:04:47Z" duration-ms="20582">
    <groups>
    </groups>
    <test started-at="2024-03-25T16:04:26Z" name="Test" finished-at="2024-03-25T16:04:47Z" duration-ms="20582">
      <class name="qaautomation.may2022.APITestAfterRefactor">
        <test-method is-config="true" signature="login()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:26Z" name="login" finished-at="2024-03-25T16:04:29Z" duration-ms="3356" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="configAPI()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:29Z" name="configAPI" finished-at="2024-03-25T16:04:35Z" duration-ms="5988" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at qaautomation.may2022.APITestAfterRefactor.configAPI(APITestAfterRefactor.java:28)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAPI -->
        <test-method is-config="true" signature="login()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:35Z" name="login" finished-at="2024-03-25T16:04:37Z" duration-ms="1337" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="dashboardAPI()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:37Z" name="dashboardAPI" finished-at="2024-03-25T16:04:38Z" duration-ms="1092" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardAPI -->
        <test-method is-config="true" signature="login()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:38Z" name="login" finished-at="2024-03-25T16:04:39Z" duration-ms="1344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="fakerTest()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:39Z" name="fakerTest" finished-at="2024-03-25T16:04:39Z" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fakerTest -->
        <test-method is-config="true" signature="login()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:39Z" name="login" finished-at="2024-03-25T16:04:40Z" duration-ms="1191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="userAPI()[pri:0, instance:qaautomation.may2022.APITestAfterRefactor@e93f3d5]" started-at="2024-03-25T16:04:40Z" name="userAPI" finished-at="2024-03-25T16:04:47Z" duration-ms="6050" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userAPI -->
      </class> <!-- qaautomation.may2022.APITestAfterRefactor -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
